frontend/TvCv19.Frontend/Domain/Repositories/ApplicationLoginRepository.cs

```diff
@@ -1,20 +1,23 @@
-﻿using System;
+﻿using Microsoft.Extensions.Configuration;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 using TvCv19.Frontend.Domain.Models;
 
 namespace TvCv19.Frontend.Domain.Repositories
 {
-    public class ApplicationLoginRepository : BaseRepository, IApplicationLoginRepository
+    public class ApplicationLoginRepository : IApplicationLoginRepository
     {
-        public Task AddApplicationLoginAsync(ApplicationLogin applicationLogin)
+        public async Task AddApplicationLoginAsync(ApplicationLogin applicationLogin)
         {
             var id = Guid.NewGuid().ToString().Replace("-", string.Empty);
-            var sql = @$"INSERT INTO `medecc`.`application-login`
-                        (id, user_name, normalized_user_name, password_hash)
-                        VALUES ('{id}', @UserName, @NormalizedUserName, @PasswordHash)";
-            return ExecuteAsync<ApplicationLogin>(sql, applicationLogin);
+            applicationLogin.Id = id;
+
+            using var context = new MedeccContext();
```

Comment:
inject MedeccContext instead of newing

Reply:
If we inject the context, it may become longer lived.  It ultimately may not matter though since the repositories should be short lived.

Reply:
yeah they should be transient or scoped i think dotnet will manage disposing as well but can keep the using just incase